database/migrations/20220411153629_initial.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('potholes', function (table) {
+      table.increments('id').primary();
+      table.string('latitude');
+      table.string('longitude');
+      table.string('description');
```

Comment:
For a `string` column type, I believe the default limit to the number of characters is 255. So if you anticipate needing more characters for this column, consider using a `text` type.